metrics-core/src/main/java/software/amazon/swage/metrics/jmx/sensor/DiskUsageSensor.java

```diff
@@ -60,8 +62,8 @@ public void sense(final MetricContext metricContext) throws SenseException
 
             long total = fileStore.getTotalSpace();
             long free = fileStore.getUsableSpace();
-            double percent_free = 100.0 * ((double)(total-free)/(double)total);
-            metricContext.record(DISK_USED, percent_free, Unit.PERCENT);
+            double percentFree = 100.0 * ((double)(total - free) / (double)total);
```

Comment:
With regard to the spaces around operators I have the same comment about style. Update to the variable name improves consistency with other code.